db/migrate/20230328162132_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name, limit: 30
+      t.string :photo
+      t.text :bio
+      t.integer :posts_count
```

Comment:
- [ ] @admirerbrown great work in your migration files, however this field should be for posts_counter instead of posts_count according to the ERD as shown below 
![image](https://user-images.githubusercontent.com/61408860/228351419-d65e4015-6bb6-4be9-bb8c-da7f3b358fa1.png)